assam/jwt/jws.py

```diff
@@ -1,15 +1,15 @@
-from typing import Tuple
-
 import python_jwt
 from jwcrypto import jwk
+from typing import Tuple, Optional
 
 _verification_alg = {
     "P-256": "ES256",
-    "secp256k1": "ES256K"
+    "secp256k1": "ES256K",
+    "P-256K": "ES256K"
 }
 
 
-def encrypt_jws(signer_key: jwk.JWK, payload: dict) -> str:
+def encrypt_jws(signer_key: jwk.JWK, payload: dict, other_headers=None) -> str:
```

Comment:
How about using `reciever_header_items` instead of `other_headers`?

Reply:
Even if the name came from the origin repository, it makes me be confused like there are multi headers.

Reply:
This value is only passed to that library. A different name seems to be more confusing.